.github/workflows/ci_tests_esp32.yml

```diff
@@ -0,0 +1,14 @@
+name: ESPRESSIF ESP32 CI Tests
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: docker://espressif/idf:latest
```

Comment:
Running on latest is little aggressive, and can potentially break our build system. Can we control this by using some release/stable version. Version which we offically support. I saw they have  image for releases like :
espressif/idf:release-v4.0

Reply:
As per our discussion, fine we will use a fixed version:v4.3